web/templates/edited_notice.hbs

```diff
@@ -1,13 +1,13 @@
 {{#if msg/local_edit_timestamp}}
-<div class="message_edit_notice auto-select tippy-zulip-delayed-tooltip" data-tippy-content="{{t 'Edited ({last_edit_timestr})' }}">
+<div class="message_edit_notice auto-select" data-tippy-content="{{t 'Last edited {last_edit_timestr}' }}">
```

Comment:
The tippy tooltip is here already, than we take it in tippy's onShow function and pass it again to tippy, Can we just remove it from here and calculate only in tippy ?
I am not aware of how much work it will take. If there is not a cleaner approach to do this, I'm also okay with this approach.

Reply:
We can pass `realm_allow_edit_history` directly here in `edited_notice.hbs`, than there will be no need of `message_edit_notice.hbs` file and changes in the `tippy.js`.

Reply:
Hey @brijsiyag , I have made the necessary changes. Could you please review it again? Thanks!

Reply:
Even I'm not sure about it.

Reply:
For now all other looks good to me.